.github/workflows/integration-test.yaml

```
@@ -48,20 +53,21 @@ jobs:
       - name: Check out xcparse repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
         with:
           repository: ably-forks/xcparse
           ref: emit-test-case-info
           path: xcparse
+          persist-credentials: false
 
       - id: get-xcparse-commit-sha
         name: Get xcparse commit SHA
         run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)